tests/x11/libcamgm.pm

```diff
@@ -19,7 +19,7 @@ my $password = $testapi::password;
 my $email    = "you\@example.com";
 
 sub restart_yast2_camgm {
-    assert_and_click("xterm");
+    assert_and_click("xterm-selected");
```

Comment:
because the needle contains the string which is specific to this test module I thought of something that is more specific but ok.